src/main/java/com/mewna/catnip/util/JsonUtil.java

```diff
@@ -38,6 +40,7 @@
 
 public final class JsonUtil {
     public static final long MAX_SAFE_INTEGER = 9007199254740991L;
+    public static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Is this just for entity de/serialization?

Reply:
yes. just copied JsonObject#mapFrom/mapTo